### ISSUE TITLE
Google Talk - XMPP

### ISSUE THREAD

**Issue by visious25** [Intika-Pidgin/Pidgin#3]
Hi, since Google disabled this option: https://myaccount.google.com/lesssecureapps
Google Talk - XMPP is unable to login.

**Comment by grimmy** [comment]
This is not the canonical repository, we (the actual pidgin developers) have nothing to do with this repository. As for the google talk situation, we made an official announcement on our official website https://pidgin.im/posts/2022-05-google-talk-shutdown/

**Comment by visious25** [comment]
Is Google Chat replacement of Google Talk? If yes, I followed the [YouTube](https://www.youtube.com/watch?v=hlDhp-eNLMU) video to get the OAuth code, but the [page](https://accounts.google.com/o/oauth2/programmatic_auth?hl=en&scope=https://www.google.com/accounts/OAuthLogin+https://www.googleapis.com/auth/userinfo.email&client_id=936475272427.apps.googleusercontent.com&access_type=offline&delegated_client_id=183697946088-m3jnlsqshjhh5lbvg05k46q1k4qqtrgn.apps.googleusercontent.com&top_level_cookie=1&from_login=1&as=enTPGhwwthn30XzTTln5RpIO6unoUlAXswhgNppUBcM&authuser=0) is never loading. In other words, make no sense to continue with Google Chat. Google Talk is just fine.

**Comment by grimmy** [comment]
The blog post I linked, links to this [page from google](https://support.google.com/talk) that states:

> Switch to Google Chat
> Important: If you try to sign in to Google Talk on or after June 16, 2022, you'll get a sign in error.
We recommend switching to [Google Chat](https://mail.google.com/chat/).

**Comment by visious25** [comment]
Yes, as I said previously I followed the [YouTube](https://www.youtube.com/watch?v=hlDhp-eNLMU) video to get the OAuth code, but the [page](https://accounts.google.com/o/oauth2/programmatic_auth?hl=en&scope=https://www.google.com/accounts/OAuthLogin+https://www.googleapis.com/auth/userinfo.email&client_id=936475272427.apps.googleusercontent.com&access_type=offline&delegated_client_id=183697946088-m3jnlsqshjhh5lbvg05k46q1k4qqtrgn.apps.googleusercontent.com&top_level_cookie=1&from_login=1&as=enTPGhwwthn30XzTTln5RpIO6unoUlAXswhgNppUBcM&authuser=0) is never loading. Which means I can't login to [purple-googlechat](https://github.com/EionRobb/purple-googlechat#readme) too.

**Comment by grimmy** [comment]
Ok, but then this is an issue for puple-googlechat and should be in their issue tracker as they're not going to see it here.

**Comment by visious25** [comment]
Where is the official repository of pidgin? I thought it was here.

**Comment by grimmy** [comment]
> Where is the official repository of pidgin? I thought it was here.

Pidgin in hosted on our own servers and uses Mercurial not git for the version control system.

There is a landing repository here on github at https://github.com/pidgin/pidgin that has links to everything.

That said, the issue with purple-googlechat needs to be filed at https://github.com/eionRobb/purple-googlechat as that is not part of Pidgin itself.

**Comment by visious25** [comment]
Thank you, I can't report Google Talk at this address https://github.com/pidgin/pidgin it's ask me pull request and not issue, but I can report Purple-Google-Chat Plugin here: https://github.com/eionRobb/purple-googlechat as well.
